docs/source/backend.rst

```diff
@@ -26,7 +26,7 @@ There are currently two implementations available.
 Availability
 ------------
 
-``"sox_io"`` backend requires C++ extension module, which is included in Linux/macOS binary distributions. This backend is not available on Windows.
+``"sox_io"`` backend requires C++ extension module. torchaudio<2.1.0 will include libsox in the Linux/macOS binary distributions, for later versions please install `libsox`. This backend is not available on Windows.
```

Comment:
backend module here refers to the part of our code that use libsox, so this statement still holds.
Can we keep this statement as-is and use `note` notation and redirect to installation ?

```suggestion
``"sox_io"`` backend requires C++ extension module, which is included in Linux/macOS binary distributions. This backend is not available on Windows.

.. note::
   Starting 2.1.0, torchaudio searches and links libsox at runtime. Please install libsox. See also `Optional Dependencies <./installation.html#optional-dependencies>`__.
```